packages/logger/src/formatters/ConsoleFormatter.ts

```diff
@@ -0,0 +1,77 @@
+import chalk from 'chalk';
+import {info, warning, error} from 'log-symbols';
+import prettyMs from 'pretty-ms';
+
+import {LogLevel, FormatEntry, Formatter} from '../types';
+
+export class ConsoleFormatter implements Formatter {
+  private startTime: [number, number];
+
+  constructor() {
+    this.startTime = process.hrtime();
+  }
+
+  next(entry: FormatEntry) {
```

Comment:
This interface feels a bit weird, I get that it's formatting the next log, but `next` almost feels like it should be yielding the  next data not parsing the next thing.

Maybe we should go with something more like `format`?